content/concepts/did-ddo.md

```diff
@@ -1,173 +1,521 @@
 ---
-title: DIDs & DDOs - Asset Identifiers & Objects
-description: Specification of Ocean asset identifiers and objects using DIDs & DDOs 
+title: DID & DDO
+description: Specification of decentralized identifiers for assets in Ocean Protocol using the DID & DDO standards.
 slug: /concepts/did-ddo/
 section: concepts
 ---
 
+**v4.0.0**
+
 ## Overview
 
-This document describes how Ocean assets follow the DID/DDO spec, such that Ocean assets can inherit DID/DDO benefits and enhance interoperability. 
+This document describes how Ocean assets follow the DID/DDO specification, such that Ocean assets can inherit DID/DDO benefits and enhance interoperability. DIDs and DDOs follow the [specification defined by the World Wide Web Consortium (W3C)](https://w3c-ccg.github.io/did-spec/).
 
-Decentralized identifiers (DIDs) are a new type of identifier that enables verifiable, decentralized digital identity. Each DID is associated with a unique entity. DIDs may represent humans, objects, and more.
+Decentralized identifiers (DIDs) are a type of identifier that enable verifiable, decentralized digital identity. Each DID is associated with a unique entity and DIDs may represent humans, objects, and more.
 
 A DID Document (DDO) is JSON blob that holds information about the DID. Given a DID, a _resolver_ will return the DDO of that DID.
 
-If a DID is the index key in a key-value pair, then the DID Document is the value to which the index key points.
-The combination of a DID and its associated DID Document forms the root record for a decentralized identifier.
+If a DID is the index key in a key-value pair, then the DID Document is the value to which the index key points. The combination of a DID and its associated DID Document forms the root record for a decentralized identifier.
```

Comment:
also confusing first sentence, what is that supposed to mean? Why is that worth mentioning it here in the overview?